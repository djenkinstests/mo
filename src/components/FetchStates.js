import React from "react";

export const Error = () => <li>Fail! </li>;

export const Loading = () => <li>Loading.... </li>;
