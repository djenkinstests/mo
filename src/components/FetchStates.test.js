import React from "react";
import { Error, Loading } from "./FetchStates";
import { shallow } from "enzyme";

describe("<Error/>", () => {
  it("renders as expected", () => {
    const component = shallow(<Error />);
    expect(component).toMatchSnapshot();
  });
});

describe("<Loading/>", () => {
  it("renders as expected", () => {
    const component = shallow(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
