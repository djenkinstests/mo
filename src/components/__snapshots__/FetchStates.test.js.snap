// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error/> renders as expected 1`] = `
<li>
  Fail! 
</li>
`;

exports[`<Loading/> renders as expected 1`] = `
<li>
  Loading.... 
</li>
`;
